lib/ios/RNNTabBarPresenter.m

```diff
@@ -54,8 +54,8 @@ - (void)mergeOptions:(RNNNavigationOptions *)newOptions currentOptions:(RNNNavig
 		[tabBarController rnn_setTabBarHideShadow:newOptions.bottomTabs.hideShadow.get];
 	}
 	
-	if (newOptions.bottomTabs.visible.hasValue) {
-		[tabBarController rnn_setTabBarVisible:newOptions.bottomTabs.visible.get];
+	if (newOptions.bottomTabs.visible.hasValue && newOptions.bottomTabs.animate.hasValue) {
```

Comment:
If `visible` has a value and `animate` doesn't, `rnn_setTabBarVisible` won't be called - This is a breaking change/bug.
I suggest we handle visibility change it two different ways:
1. Keep current behaviour as is - Visible has value and animate doesn't
2. Both visible and animate have values

Once you make this change we can move forward and approve this PR. Thanks

Reply:
Changed :)
However, there's an Android test failed:
```Failed to install the following Android SDK packages as some licences have not been accepted.```
How should I fix this?